### DISCUSSION TITLE
2024/12/emacs-everywhere/

### DISCUSSION THREAD

**Discussion by SouthFox-D** [SouthFox-D/SouthFox-D.github.io#99] [博客评论]
https://blog.southfox.me/2024/12/emacs-everywhere/

**Comment by midas-brave** [comment]
折腾就是第一生产力

**Comment by kyoshiraishi** [comment]
安卓能折腾的东西还是多啊~ iOS user就安心待在beorg的小窝里了，作为org capture的移动端输入口也还能接受。
